ios/LayoutReanimation/REAAnimationsManager.m

```diff
@@ -522,7 +522,11 @@ - (void)viewDidMount:(UIView *)view withBeforeSnapshot:(nonnull REASnapshot *)be
   }
 
   if (_hasAnimationForTag(viewTag, @"sharedElementTransition") && [type isEqual:@"entering"]) {
-    [_sharedTransitionManager notifyAboutNewView:view];
+    if ([type isEqual:@"entering"]) {
+      [_sharedTransitionManager notifyAboutNewView:view];
+    } else if ([type isEqual:@"layout"]) {
+      [_sharedTransitionManager notifyAboutViewLayout:view withViewFrame:frame];
+    }
```

Comment:
I don't understand why we check if `[type isEqual:@"layout"]` here since we already checked that `[type isEqual:@"entering"]`.

Reply:
Right, my bad